shell/platform/android/test/io/flutter/plugin/editing/InputConnectionAdaptorTest.java

```diff
@@ -171,6 +174,44 @@ public void testPerformContextMenuAction_paste() {
     assertTrue(editable.toString().startsWith(textToBePasted));
   }
 
+  @Test
+  public void testCommitContent() throws JSONException {
```

Comment:
Heads up that this test appears to be failing

```

io.flutter.plugin.editing.InputConnectionAdaptorTest > testCommitContent FAILED
    Wanted but not invoked:
    dartExecutor.send(
        <Capturing argument>,
        <Capturing argument>,
        isNull()
    );
    -> at io.flutter.embedding.engine.dart.DartExecutor.send(DartExecutor.java:223)

    However, there was exactly 1 interaction with this mock:
    dartExecutor.setMessageHandler(
        "flutter/textinput",
        io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler@18698c6a
    );
    -> at io.flutter.plugin.common.MethodChannel.setMethodCallHandler(MethodChannel.java:146)
        at io.flutter.embedding.engine.dart.DartExecutor.send(DartExecutor.java:223)
        at io.flutter.plugin.editing.InputConnectionAdaptorTest.testCommitContent(InputConnectionAdaptorTest.java:203)
```

Reply:
I'm aware of this, the error hasn't really helped me to figure out why its failing. No idea what needs to change to get it to work.